Add publicIp & vNic mgmt features

jihoon-seo opened the pull request.

seokho-son commented:
@jihoon-seo 
Go file name convention 을 고려하면, file name에 camelCase를 사용하지 않는 것이 일반적인 것 같습니다. ( '_' 도 그리 좋은 방식을 아닌 것 같지만 .. )
참고) https://www.quora.com/What-is-the-best-naming-convention-of-keeping-source-files-in-a-programming-language
동의하신다면 ~ PR 업데이트 부탁 드립니다.

jihoon-seo commented:
다음과 같이 rename 하였습니다.
- renamed:    mcism_resource_manage_publicIp.go -> mcism_resource_manage_publicip.go
- renamed:    mcism_resource_manage_securityGroup.go -> mcism_resource_manage_securitygroup.go
- renamed:    mcism_resource_manage_sshKey.go -> mcism_resource_manage_sshkey.go
- renamed:    mcism_resource_manage_vNic.go -> mcism_resource_manage_vnic.go
